MissionHotpointTask hotpoint_task
---
bool result
# for debugging usage
uint8 cmd_set
uint8 cmd_id
uint32 ack_data